packages/http/tools/e2e-tests/features/statusCode.feature

```diff
@@ -0,0 +1,42 @@
+Feature: statusCode decorator
+
+The statusCode decorator allows set response status code
+
+  Background: Having a container
+    Given a container
+
+    Rule: statusCode decorator allows set response status code
+      Scenario: Response status code is correctly set
+
+        Given a warrior controller with statusCode decorator for <method> method
```

Comment:
```suggestion
        Given a warrior controller with NO_CONTENT statusCode decorator for <method> method
```
What about this? This way it's more explicit

Reply:
Done